testfiles/app_admin/app_admin.js

```diff
@@ -0,0 +1,34 @@
+import { Given, When, Then, Before, After, And } from 'cypress-cucumber-preprocessor/steps';
+import { ADMIN_USER } from '../../support/constants';
+
+Before(() => {
+  cy.loginCognito(ADMIN_USER).then((idToken) => cy.wrap(idToken).as('idToken'));
+  cy.visit('/');
+
+  cy.get('[data-testid=create-author-button]', { timeout: 10000 }).click();

```

Comment:
ser ut som man kan sette default timout globalt også: https://docs.cypress.io/guides/references/configuration.html#Timeouts
(hvis det er noe vi vil da)

Reply:
Jeg skal se på det, det er en del venting i frontend som jeg må ta høyde for.